ReactNative/ReactTwitter/core/service/push-notifications-service.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import { AsyncStorage } from 'react-native'
+var PushNotification = require('react-native-push-notification')
+
+const STORAGE_KEY = '@PushNotificationToken:key'
+
+class PushNotificationsService {
+  static init () {
+    PushNotification.configure({
+
+      // (optional) Called when Token is generated (iOS and Android)
+      onRegister (token) {
+        console.log('TOKEN:', token)
+        AsyncStorage.setItem(STORAGE_KEY, token.token)
+      },
+
+      // (required) Called when a remote or local notification is opened or received
+      onNotification (notification) {
+        console.warn('NOTIFICATION:', notification)
```

Comment:
Changed with `Alert.alert()`, and it seems that the callback is actually called when opening a notification (see the updated gifs). Thanks 😂